include/library/spdm_common_lib.h

```diff
@@ -66,6 +66,7 @@ typedef enum {
     /* Certificate info*/
 
     LIBSPDM_DATA_LOCAL_PUBLIC_CERT_CHAIN,
+    LIBSPDM_DATA_LOCAL_SLOT_COUNT,
```

Comment:
please add comment: // reserved.

Reply:
Thanks. I have added it.

Reply:
thank you.
Since reserved is not put in the same line, I feel we should say "LIBSPDM_DATA_LOCAL_SLOT_COUNT is deprecated. Do not use it. Reserved only."

Reply:
Thanks. I have added it.